### DISCUSSION TITLE
Example import text for create message app

### DISCUSSION THREAD

**Discussion by Jamessigur** [Budibase/budibase#8268] [Help]
![Messenger_201902_01](https://user-images.githubusercontent.com/105172391/195930387-73de0401-a738-4e5b-b2f9-8fc621d24fa8.png)

Hello everyone, I ask you to help implement a possibly simple task in BB. I have a text file that contains, for example:
"User1: Hi
User2: Hello
.....etc."
There is a background image (example in the screenshot). How to automatically substitute the text User1 in Text.User1 and User2 in Text.User2 on the screenshot?

**Comment by andz-bb** [comment]
Hey @Jamessigur, I'm not sure I understand what you're trying to do here. Could you explain in more detail and, if you have one, send an app export to support@budibase.com so I can have a look? Thanks 😊
